src/PrestaShopBundle/EventListener/MethodOverrideListener.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace PrestaShopBundle\EventListener;
+
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+
+/**
+ * Listens to the request event to override the HTTP method if needed.
+ */
+class MethodOverrideListener
```

Comment:
You shouldn't need this, Symfony is supposed to handle this already It may need to be re-enabled though https://github.com/PrestaShop/PrestaShop/blob/7308a67174f0876c2a530f4a6082a4270c3abe8d/admin-dev/index.php#L75